support-frontend/assets/pages/[countryGroupId]/oneTimeCheckout.tsx

```diff
@@ -831,6 +848,17 @@ function OneTimeCheckoutComponent({
 								})}
 							</RadioGroup>
 						</FormSection>
+						<CoverTransactionCost
+							transactionCost={coverTransactionCost}
+							transactionCostCopy={transactionCostCopy}
+							onChecked={(check) => {
+								setCoverTransactionCost(check);
+							}}
+							transactionCostAmount={simpleFormatAmount(
+								currency,
+								finalAmount ? finalAmount : 0,
```

Comment:
Is showing "I’d like to add a further £0" the copy we want? I guess it's an edge case

Reply:
Yep, I checked that and the current checkout does this, so presumed was ok.
![image](https://github.com/user-attachments/assets/12b14589-9ade-4b32-8ae3-14ae82e4ba08)